frontend/src/api/useDeleteData.js

```diff
@@ -0,0 +1,28 @@
+import { useEffect, useState } from 'react';
+
+const useDeleteData = (url) => {
+  const [status, setStatus] = useState(null);
+  const [error, setError] = useState(null);
+  const [isLoading, setIsLoading] = useState(false);
+
+  useEffect(() => {
+    (async function sendDeleteRequest() {
+      try {
+        setIsLoading(true);
+        const response = await fetch(`${url}`, { method: 'DELETE' });
+        setStatus(response.status);
```

Comment:
Maybe we should replace the status or add isSuccess flag. 

With isSuccess flag, we can simply show something in the component instead of checking status code first.